cypress/e2e/changeAccount.cy.js

```diff
@@ -3,7 +3,6 @@ describe('Change Account', () => {
     it('Change Account', () => {
       cy.visit('https://barrigareact.wcaquino.me')
```

Comment:
Instead of using the URL in the spec file, you could use it in the cypress.config.js.
e.g, cy.visit('/')